_posts/2023-05-20-introduction-juanita.md

```diff
@@ -0,0 +1,103 @@
+
+## Verse 3: A Blend of Passions
+
+Apart from my scientific Python work, I find comfort and happiness in music (hence the weird way in which I decided to write this blogpost 🤣).  Being a singer brings me immense inspiration and energy, serving as a joyful escape from the multitude of tasks and responsibilities I undertake. It prevents me from becoming overwhelmed by the various aspects of my involvement. I embrace my geeky side 🤓, always eager to learn and expand my knowledge (hence the PhD) but I'm also deeply passionate about art, especially music, and I strive to be involved in both worlds. However, above all, I care deeply about communities and find great pleasure in dedicating myself to the projects I support.
+
+## Bridge: A New Melody - Joining pyOpenSci
+
+<figure style="width: 250px;margin-top:0.4em;margin-bottom:0" class="align-left">
+    <a href="/images/introduction-juanita/leah-pyopensci.jpeg">
+    <img src="/images/introduction-juanita/leah-pyopensci.jpeg" style="max-width:90%" alt="Two women wearing sunglasses, enjoying a sunny day.">
+    </a>
+    <figcaption> I was thrilled to recently meet Leah, the Executive Director of pyOpenSci, in person. I'm excited to work with her on this new journey and eager to learn from her experience with open source communities.
+    </figcaption>
+</figure>
+
+Recently, I joined pyOpenSci, an organization that resonates deeply with my values. Their commitment to enhancing the scientific Python ecosystem through a rigorous [peer review process](https://www.pyopensci.org/software-peer-review/about/intro.html) aligns perfectly with my own aspirations. This process not only recognizes the hard work of scientists contributing to open Python tools, but also promotes standardization and increased visibility for these packages. I am excited to bring my past experiences to the table, helping to strengthen community building and outreach within this incredible project while eagerly learning from the talented individuals who are part of it.
+
+## Outro: Gratitude
+
+As a community manager at both Scientific Python and pyOpenSci, I have witnessed the wonders of the Scientific Python ecosystem. I have encountered countless remarkable individuals and witnessed the incredible projects they have created using the available tools. In my opinion, unifying the community and promoting project integration is the key to unlocking the full potential of this ecosystem. It's not just about coding; it's about coming together, sharing knowledge, and supporting one another. I am grateful for the opportunity to contribute to this vision and help build an extraordinary community around the scientific Python ecosystem while walking this journey alongside amazing people. 🫶
```

Comment:
@juanis2112 this is really good. i wanted to clarify the position because again we have a full time community manager coming on board so i don't want any confusion as she has been hired. So i just adjusting the language here and also made scientific Python lowercase to avoid confusion (which already exists!) because pyOpenSci, the scientific Python broader community and the Scientific Python project. If you are ok with this we can merge this and then merge the entire blog. just let me know.